### ISSUE TITLE
Crashes on load everytime.

### ISSUE THREAD

**Issue by Twizzer** [GriefMoDz/better-status-indicators#55]
Crashes on every launch and unrecoverable.

**Comment by GriefMoDz** [comment]
I'm sorry to hear that. Are you able to clarify which client mod you're using? Can you also please provide me with a list of your installed plug-ins? The more information I can get, the better it is. Thanks.

**Comment by Twizzer** [comment]
I'm using Kernel and I got it to not crash issue I'm having now is the text for the toggles even with all plugins off and themes off is blank.

**Comment by GriefMoDz** [comment]
Glad to hear that it's no longer causing a crash. I'm surprised to hear that the settings switches are now showing as blank for you. What release channel are you running Kernel on?

**Comment by Absumm** [comment]
Hi! this still crashes on replugged/powercord

**Comment by GriefMoDz** [comment]
> Hi! this still crashes on replugged/powercord

@Absumm Do you have any errors that you can show me? You can open your DevTools console (`CTRL+SHIFT+I` and select the "Console" tab at the top) to check this.

**Comment by Absumm** [comment]
![image](https://user-images.githubusercontent.com/56665292/192099588-3528bd55-584f-4f77-84dd-7d32e4d8ede8.png)
 
 it'll work for a sec and then crash
 
bottom-most error is for bdCompat, which I doubt is causing the issues

**Comment by GriefMoDz** [comment]
Are you using Replugged on the stable release channel?

**Comment by Absumm** [comment]
do you mean on stable discord client?

prolly a dumb question but fairly tired.

**Comment by GriefMoDz** [comment]
> do you mean on stable discord client?
> 
> prolly a dumb question but fairly tired.

Yes, that's correct.

**Comment by Absumm** [comment]
no, currently I'm using canary client because I was just used to using the canary client from original powercord.

assuming you no longer update for canary?

**Comment by GriefMoDz** [comment]
Huh, that's odd. I'm also using Canary and I don't receive any of those errors, nor does Better Status Indicators crash for me. Is there any chance that you could send over your debug info under the "Updater" tab of "User Settings"?
